src/classdef.cpp

```diff
@@ -3646,7 +3646,8 @@ void ClassDefImpl::mergeMembers()
 
   m_impl->membersMerged=TRUE;
   //printf("  mergeMembers for %s\n",name().data());
-  bool inlineInheritedMembers = Config_getBool(INLINE_INHERITED_MEMB);
+  static bool inlineInheritedMembers = Config_getBool(INLINE_INHERITED_MEMB);
+  static bool extractPrivate         = Config_getBool(EXTRACT_PRIVATE);
```

Comment:
Why not `static const bool`?

Reply:
All other places where a similar construct was used just used `static bool` as well, so I did it for consistency.
Feel free to create a pull request (maybe look at all `Config_get...` methods.

Reply:
I agree consistency is best here, so I'll do as you suggest and send a pull request to change them all.

Thanks for the speedy fix.